Gemfile

```diff
@@ -1,32 +1,26 @@
 source "https://rubygems.org"
 
-ruby '1.9.3'
+ruby '2.7.2'
 
 gem 'thin'
 gem "sequel",  "~> 4.2"
 gem "rack-robustness", "~> 1.1"
-gem "rack-timeout"
-gem "rake"
-gem 'sprockets', '~> 2.0'
+gem "rack-timeout", "~> 0.6"
+gem "rake", "~> 13.0"
+gem 'sprockets', '~> 4.0'
 gem 'sinatra', '~> 1.4'
 gem 'wlang', '~> 2.3'
 
-gem "alf-core",   :git => "git://github.com/alf-tool/alf-core.git"
-gem "alf-doc",    :git => "git://github.com/alf-tool/alf-doc.git"
-gem "alf-sql",    :git => "git://github.com/alf-tool/alf-sql.git"
-gem "alf-sequel", :git => "git://github.com/alf-tool/alf-sequel.git"
-gem "alf-rack",   :git => "git://github.com/alf-tool/alf-rack.git"
+gem "alf-core",   "~> 0.16.3"
+gem "alf-doc",    "~> 0.16.3"
+gem "alf-sql",    "~> 0.16.3"
+gem "alf-sequel", "~> 0.16.3"
+gem "alf-rack",   "~> 0.16.3"
 
```

Comment:
Ok, but are we sure there are no non-commited changes on master of those git repos?

Reply:
@blambeau I thought it was preferable to fix the version so not to be surprised by new changes?

Reply:
Yeah, but I would not be surprised that try-alf uses non released features. Could you maybe check (on alf-core, alf-rack and alf-sql) whether there are unreleased changes (compare v0.16.3..master and see how many commits)?

Reply:
Ok

Reply:
@blambeau 21 commits.